tsconfig.json

```diff
@@ -4,6 +4,13 @@
 		"declaration": false,
 		"emitDecoratorMetadata": true,
 		"experimentalDecorators": true,
+		"lib": [
+			"dom",
+			"es5",
+			"es2015.iterable",
+			"es2015.symbol",
+			"es2015.symbol.wellknown"
+		],
```

Comment:
It was in all of the other similar PRs which is why it caught my attention, e.g. https://github.com/dojo/shim/pull/76/files#diff-e5e546dd2eb0351f813d63d1b39dbc48R23 , so mostly I noticed that this and i18n were inconsistent with the earlier PRs.